invenio_records_resources/translations/messages.pot

```diff
@@ -1,36 +1,40 @@
 # Translations template for invenio-records-resources.
-# Copyright (C) 2022 CERN
+# Copyright (C) 2023 CERN
```

Comment:
Shelvable:
@mb-wali I recall we chat about this happening. If we run the tests locally, the `.pot` files get modified. We end up pushing changes every PR... What was the outcome of it in the end?

Reply:
could you remove this file from the commit?